api/api/specs/routers.yaml

```diff
@@ -401,9 +401,12 @@ paths:
           content:
             application/json:
               schema:
-                type: "array"
-                items:
-                  $ref: "#/components/schemas/Event"
+                type: "object"
+                properties:
+                  events:
+                    type: "array"
+                    items:
+                      $ref: "#/components/schemas/Event"
```

Comment:
Somehow the original schema doesn't align with the responses I'm getting from the API, so I'm aligning this with that.